### ISSUE TITLE
Suggestion to not use camelcase for span tags

### ISSUE THREAD

**Issue by Sam-Kruglov** [AxonFramework/extension-tracing#45]
Currently, we have this:
https://github.com/AxonFramework/extension-tracing/blob/80d3633bc280efd7c9c62822327508c4b7166dfa/tracing/src/main/java/org/axonframework/extensions/tracing/SpanUtils.java#L36-L40

I feel this would be better:
```java
private static final String TAG_AXON_ID = "axon.message.id";
private static final String TAG_AXON_AGGREGATE_ID = "axon.message.aggregate.id";
private static final String TAG_AXON_MESSAGE_TYPE = "axon.message.type";
private static final String TAG_AXON_PAYLOAD_TYPE = "axon.message.payload.type";
private static final String TAG_AXON_MESSAGE_NAME = "axon.message.name";
```

My reasons:
* cleaner look
* Jaeger UI displays them in lower case
![Screenshot 2020-04-22 at 18 45 27](https://user-images.githubusercontent.com/23556336/80003269-71289780-84c9-11ea-9238-364084e13985.png)

**Comment by smcvb** [comment]
Gotcha, that's not overly nice indeed. Guessing it differs per Open Tracing (UI) implementation, but still, not to nice. However I do not feel to much for the dot-notation you are sharing. 
Make it seem like there is another level on `axon.message` tags, which is `aggregate` and `payload`, whilst actually those belong on the message level directly.

What would you say about using dashes or underscore per chance?

**Comment by Sam-Kruglov** [comment]
There's gotta be some common patter how people name these tags, personally, I like dashes more :)

I don't think it's a problem to make it seem like more nesting, message.aggregate.id makes sense to me as well

**Comment by smcvb** [comment]
> There's gotta be some common patter how people name these tags,..

I was assuming the same, but haven't been able to find a decisive answer. I did find this exert from [Open Tracing's "Best Practices" page](https://opentracing.io/docs/best-practices/) though, under header *Continuing or Starting a Trace from an Incoming Request*: 

> The OpenTracing API does not dictate how applications name the spans.

So...guessing there's no decisive answer for now. However:

>  I like dashes more :)

I'd be leaning towards dashes too given the current outcome. Hence I'll do a push for this and close this issue as resolved afterwards.

**Comment by smcvb** [comment]
Closing this issue as it has been resolved in commit d59b13b.
